### ISSUE TITLE
JavaSctipt

### ISSUE THREAD

**Issue by wmantly** [himleyb85/recession_survivor#86]
Please do not make anything depend on JavaScript, its very bad practice. Make everything work with out JS first, then add JS to enhance the user experience. 
For this project it may not be that important, but out in the wild( where this project may wind up one day ) its very important;
1. Search engine crawlers cant use JS and cant index pages properly that need JS to function.
2. Some people dont have JS or cant use it. For example;
   - People with disabilities use special browsers that dont work well with JS and usually just disable it. 
   - Some organizations and people see JS as a security risk and force it to be disabled.( ie Tor )
3. Older browsers will not run modern JS correctly.

Bottom line, JavaScript should always gracefully fail.  
http://www.sitepoint.com/progressive-enhancement-graceful-degradation-basics/
